org-formation/_scp.yaml

```diff
@@ -126,3 +161,18 @@
               - 'route53domains:RegisterDomain'
               - 'route53domains:TransferDomain'
             Resource: '*'
```

Comment:
Maybe I'm missing something... There are still some domains that need to be transferred to sageIT, and we need to register domains in sageIT. Does not this rule prohibit this? Do we need a condition here to give admins (or some role) the right to do register/transfer domains?

Reply:
yes, your understanding of how this SCP works is correct @xschildw.  However you also need to consider how it's used.  The application of this SCP is in [organizations.yaml](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/organization.yaml).  We are applying this to nearly all of our accounts except the `SageITAccount`.